packages/devtools/test/composite.test.ts

```diff
@@ -520,7 +530,70 @@ describe('composite', () => {
     expect(params).not.toEqual(third)
   })
 
-  test.todo('Composite.create()')
+  describe('Composite.create()', () => {
+    const timeout = 60000
+    jest.setTimeout(timeout)
+
+    const ceramic = new CeramicClient('http://localhost:7007')
+    const resolver = new Resolver({
+      ...KeyDidResolver.getResolver(),
+    })
+    const did = new DID({
+      resolver: resolver,
+      provider: new Ed25519Provider(random.randomBytes(32)),
+    })
+    ceramic.did = did
+
+    beforeAll(async () => {
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-call
+      await setup({
+        command:
+          'rm -rf ./test/statestore && ceramic daemon --network inmemory --state-store-directory ./test/statestore',
+        debug: true,
+        launchTimeout: timeout,
+        port: 7007,
+      })
+    })
+
+    afterAll(async () => {
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-call
+      await teardown()
+    })
+
+    test(
+      'creates a new composite from valid schema',
+      async () => {
+        await did.authenticate()
+        const composite = await Composite.create({
+          ceramic: ceramic,
+          schema: compositeSchemaWithProfiles,
+          metadata: {
+            controller: ceramic.did.id,
+          },
+        })
+        expect(composite.hash).not.toBeFalsy()
```

Comment:
Can you check the actual composite definition contents please? Things like having all the models, maybe based on `composite.toParams()`?